src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,23 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i <= 2) {
```

Comment:
also, think about how you should change this condition according to the previous comment suggestion

Reply:
not fixed